cursor_test.go

```diff
@@ -104,23 +104,69 @@ func TestCursorSelectError(t *testing.T) {
 
 	// Try to select a column that doesn't exist.
 	c := r.Select("notfound")
-	
+
 	var hasNext bool
 	for c.Next() {
 		hasNext = true
 	}
-	
+
 	if hasNext {
 		t.Errorf("Next returned true, expected false")
 	}
-	
+
 	err = c.Err()
 	if err == nil {
 		t.Errorf("Expected error")
 	}
 	if err.Error() != "no field with name: notfound" {
 		t.Errorf("Unexpected error: %s", err.Error())
 	}
-	
-	
+
+}
+
+func TestCursorSelectStripe(t *testing.T) {
+	r, err := Open("./examples/demo-11-none.orc")
+	if err != nil {
+		t.Fatalf("Expected no error, got %v", err)
+	}
+
+	expectedStripes := 385
+
+	numStripes, err := r.NumStripes()
+	if err != nil {
+		t.Fatalf("Expected no error, got %v", err)
+	}
+
+	if numStripes != expectedStripes {
+		t.Fatalf("Expected %d stripes, got %d", expectedStripes, numStripes)
+	}
+
+	cols := r.Schema().Columns()
+	c := r.Select(cols...)
+
+	err = c.SelectStripe(numStripes - 1)
+	if err != nil {
+		t.Fatalf("Expected no error, got %v", err)
+	}
+
+	var row []interface{}
+	for c.Next() {
+		row = c.Row()
+	}
+
+	expectedLastRow := []interface{}{
+		int64(1920800),
+		"F",
+		"U",
```

Comment:
Uhhh...

Reply:
This is the last row of data in the sample Apache ORC files. Sadly it comes off as a little aggressive.